modules/detection__enumerate_services/main.py

```diff
@@ -6,13 +6,13 @@
 
 module_info = {
     # Name of the module (should be the same as the filename)
-    'name': 'enum_monitoring',
+    'name': 'detection__enumerate_services',
```

Comment:
change `detection__enumerate_services` and all it's other references to `detection__enum_services`